docs/transfers.rst

```diff
@@ -0,0 +1,296 @@
+   :scale: 100 %
+   :alt: Different ways of sending a transfer in IOTA.
+
+   API commands for sending transfers.
+
+Let's look at some code snippets on how to perform the above with an imaginary
+bundle that has 3 fictional transactions.
+
+1. Level Padawan
+~~~~~~~~~~~~~~~~
+The easiest and most convenient way is to use :py:meth:`~Iota.send_transfer`
+extended API method. You still need to create the transactions yourself
+with :py:class:`ProposedTransaction`.
+
+.. code-block::
+
+    from iota import Iota, ProposedTransaction, Address
+
+    api = Iota('https://nodes.devnet.iota.org:443')
+
+    fictional_transactions = [
+        ProposedTransaction(
+                address=Address(b'FIRSTRANDOMADDRESS'),
+                value=0,
+                # You could add a tag or message here too!
+        ),
+        ProposedTransaction(
+                address=Address(b'SECONDRANDOMADDRESS'),
+                value=0,
+        ),
+        ProposedTransaction(
+                address=Address(b'THIRDRANDOMADDRESS'),
+                value=0,
+        )
+    ]
+
+    imaginary_bundle = api.send_transfer(
+        transfers=transactions
+    )['bundle']
+
+As all API methods in PyOTA, :py:meth:`~Iota.send_transfer` also returns
+a ``dict``. The ``bundle`` key holds the value of :py:class:`Bundle`.
+
+It's important to note, that for value transfers, you will need your seed as well.
+:py:meth:`~Iota.send_transfer` will look for ``input addresses`` to fund outgoing
+transactions in the bundle, and auto-generate an unused ``change address`` if
+there is a remainder amount of tokens. It will also take care of finalizing the
+bundle and signing the necessary input transactions.
+
+2. Level Obi-Wan
+~~~~~~~~~~~~~~~~
+Instead of :py:meth:`~Iota.send_transfer`, you can use the combination of
+:py:meth:`~Iota.prepare_transfer` and :py:meth:`~Iota.send_trytes` to achieve
+the same result.
+
+.. code-block::
+
+    from iota import Iota, ProposedTransaction, Address
+
+    api = Iota('https://nodes.devnet.iota.org:443')
+
+    transactions = [
+        ProposedTransaction(
+                address=Address(b'FIRSTRANDOMADDRESS'),
+                value=0,
+        ),
+        ProposedTransaction(
+                address=Address(b'SECONDRANDOMADDRESS'),
+                value=0,
+        ),
+        ProposedTransaction(
+                address=Address(b'THIRDRANDOMADDRESS'),
+                value=0,
+        )
+    ]
+
+    prepared_trytes = api.prepare_transfer(
+        transfers=transactions
+    )['trytes']
+
+    imaginary_bundle_trytes = api.send_trytes(
+        trytes=prepared_trytes
+    )['trytes']
+
+A difference here is that the end result, ``imaginary_bundle_trytes`` is a list
+of :py:class:`TransactionTrytes`, and not a :py:class:`Bundle` object.
+
+3. Level Yoda
+~~~~~~~~~~~~~
+Being the master Jedi of the PyOTA universe means that you know the most about
+the force of low-level API methods. Use it wisely!
```

Comment:
```suggestion
the force of low-level API methods. Use it wisely!

.. tip::
  You generally won't need to split out the process explicitly like this in your application code, but it is useful to understand what :py:meth:`~Iota.send_transfer` does under-the-hood, so that you are better-equipped to troubleshoot any issues that occur.
```